google/cloud/spanner/polling_policy.h

```diff
@@ -24,15 +24,58 @@ namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
 
+/**
+ * Control the Cloud Spanner C++ client library behavior with respect to polling
+ * on long running operations.
+ *
+ * Some operations in Cloud Spanner return a `google.longrunning.Operation`
+ * object. As their name implies, these objects represent requests that may take
+ * a long time to complete, in the case of Cloud Spanner some operations may
+ * take tens of seconds or even 30 minutes to complete.
+ *
+ * The Cloud Spanner C++ client library models these long running operations
+ * as a `google::cloud::future<StatusOr<T>>`, where `T` represents the final
+ * result of the operation. In the background, the library polls the service
+ * until the operation completes (or fails) and then satisfies the future.
+ *
+ * This class defines the interface for policies that control the behavior of
+ * this polling loop.
+ *
+ * @see https://aip.dev/151 for more information on long running operations.
+ */
 class PollingPolicy {
  public:
   virtual ~PollingPolicy() = default;
 
+  /**
+   * Return a copy of the current policy.
+   *
+   * This function is called at the beginning of the polling loop. Policies that
+   * are based on relative time should take a timestamp when this function is
```

Comment:
"take a timestamp" may be a little unclear. Maybe "restart their timer" or something? Not sure of the best wording.

Reply:
Fixed, thanks.